save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/repository/SandboxUserRepository.kt

```diff
@@ -23,5 +24,58 @@ class SandboxUserRepository(
                 mapOf("name" to name),
                 Long::class.java
             )
-                .orNotFound { "There is not user with name $name" }
+                .orNotFound { "There is no user with name $name" }
+
+    /**
+     * @param name
+     * @return user or null if no results have been found
+     */
+    @Transactional
+    fun findByName(name: String): User? {
```

Comment:
How are we solving the problem with user data? Is this data duplicated to `save_cloud_sandbox` schema or does sandbox share data with backend?

Reply:
sandbox uses `save_cloud.user` -- table from backend

Reply:
https://github.com/saveourtool/save-cloud/pull/1310#discussion_r994423453